src/_guides/libraries/library-tour.md

```diff
@@ -536,18 +525,20 @@ Although Map doesn’t implement Iterable, you can get Iterables from it
 using the Map `keys` and `values` properties.
 </div>
 
-Use `isEmpty` to check whether a list, set, or map has no items:
+Use `isEmpty` or `isNotEmpty` to check whether a list, set, or map has items:
```

Comment:
I wonder if we should add a comment explaining why the core library even has `isNotEmpty`. Something like "Practice showed that developers use `!list.isEmpty` a lot and so — after careful consideration — we decided to add `isNotEmpty`. It's redundant, but developers find it handy, readable and pragmatic."

cc @kwalrath 

Consider this non-blocking. I'd love to get Kathy's opinion once she gets back.

Reply:
SGTM, I'll add this to a list of topic to review with Kathy when she returns.

Reply:
I don't want to add a comment this to the tour, since it generally tries to be as succinct as possible. If there's a place that talks about designing APIs (e.g. Effective Dart), _that_ might be an appropriate place to mention this.

Reply:
There is _some_ form of justification given in this Effective Dart Usage Dont': https://www.dartlang.org/guides/language/effective-dart/usage#dont-use-length-to-see-if-a-collection-is-empty

Shall I make "Use `isEmpty` or `isNotEmpty`" a link to this entry?